scripts/diff_jsons.py

```diff
@@ -15,12 +16,22 @@
     parser.add_argument("first_json", help="first JSON to compare")
     parser.add_argument("second_json", help="second JSON to compare")
     parser.add_argument("--significant-digits", type=int, default=5, help="number of significant digits to use when comparing numeric values")
-    parser.add_argument("--exclude-paths", nargs="+", action=ExtendOverwriteDefault, help="list of paths to exclude from consideration when performing a diff", default=["root['generated_by']['version']"])
+    parser.add_argument("--exclude-paths", nargs="+", action=ExtendOverwriteDefault, help="list of paths to exclude from consideration when performing a diff", default=["root['generated_by']", "root['meta']['updated']"])
     parser.add_argument("--exclude-regex-paths", nargs="+", action="extend", help="list of path regular expressions to exclude from consideration when performing a diff")
     parser.add_argument("--ignore-numeric-type-changes", action="store_true", help="ignore numeric type changes in the diff (e.g., int of 1 to float of 1.0)")
 
     args = parser.parse_args()
 
+    # Test for most fatal errors in regex path usage
+    # Exclude regexes should never match `'`, otherwise the diff is always going to pass
+    for regex in args.exclude_regex_paths or []:
```

Comment:
_nit_

Adding `or []` (e596b501a73409fde7ea71a24f1c4877a82f442a) doesn't seem useful:

- We don't run mypy/pyright on this file
- A better solution would be to set `default=[]` on `add_argument()`

Reply:
Just because we don't run mypy doesn't mean that we shouldn't prevent type errors from happening?

Why should we have to change the defaults?

Reply:
> Why is changing the defaults better?

Setting the default up top allows for a valid assumption that `args.exclude_regex_paths` is always iterable. Otherwise we would have to do another `args.exclude_regex_paths or []` if using it as an iterable somewhere else in the code for whatever reason.